test/functional/services/dashboard/panel_actions.js

```diff
@@ -140,7 +140,17 @@ export function DashboardPanelActionsProvider({ getService, getPageObjects }) {
         panelOptions = await this.getPanelHeading(originalTitle);
       }
       await this.customizePanel(panelOptions);
-      await testSubjects.setValue('customDashboardPanelTitleInput', customTitle);
+      if (customTitle.length === 0) {
+        if (browser.isW3CEnabled) {
+          const input = await testSubjects.find('customDashboardPanelTitleInput');
```

Comment:
It shouldn't. It is specific to this case where what firefox and chrome has a unique working solution: clearing via keyboard works only in FF (`browser.isW3CEnabled` case), clearing with 'h\b' is only working way in Chrome (JSON wire now, but I guess it will stop working soon as Chrome is W3C by default)